<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>640</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Listado videojuegos</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="lbl_listado">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>841</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Showcard Gothic</family>
     <pointsize>12</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Listado de videojuegos (List Widget)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="lst_listado">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>841</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_numero_juegos">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>590</y>
     <width>200</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
